prediction_market_agent/agents/utils.py

```diff
@@ -29,6 +29,7 @@ class AgentIdentifier(str, Enum):
     MICROCHAIN_AGENT_OMEN_LEARNING_2 = "general-agent-2"
     MICROCHAIN_AGENT_OMEN_LEARNING_3 = "general-agent-3"
     MICROCHAIN_AGENT_STREAMLIT = "microchain-streamlit-app"
+    MICROCHAIN_AGENT_OMEN_WITH_GOAL_MANAGER = "general-agent-4-with-goal-manager"
```

Comment:
```suggestion
    MICROCHAIN_AGENT_OMEN_WITH_GOAL_MANAGER = "trader-agent-4-with-goal-manager"
```

general agents above are the ones that are learning, this one (based on trader prompt) should be only trader, if that's right, can we distinguish it?

Reply:
Ah true. How about `trader-agent-0-with-goal-manager`, as there are no other `trader-agent-*`s in the list?

Reply:
Sure!